src/main/java/wolox/training/models/Book.java

```diff
@@ -9,7 +9,8 @@
 
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
-    private long id;
+    @Column(nullable = false)
```

Comment:
Not necessary as given it's @Id, it will be primary key and therefore not nullable.